doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,473 @@
+This ensures that if a manifest belonging to one local domain (for example, the application domain) attempts to declare components that span into areas belonging to another local domain (for example, the radio domain or, more critically, the Secure Domain), the system can detect and block this behavior.
+
+SUIT topologies
+*****************
+
+The supported SUIT topologies are the following.
+
+Invocation path (normal booting)
+================================
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_invocation_topology.png
+
+   SUIT topology for nRF54H20 invocation path in the most complex predicted case
+
+In the event of a device cold boot, following the standard invocation procedure, the SDFW will first execute the validation, loading, and invocation sequences defined in the top manifest from Nordic Semiconductor.
+Next, it will execute these sequences as specified in the root manifest.
+
+The invocation of Nordic-related SUIT elements is fully controlled by SUIT manifests signed by Nordic Semiconductor.
+This approach ensures that entities other than Nordic cannot manipulate the order of execution of Nordic-controlled elements.
+
+Both the Nordic top and root manifests control the boot process of their respective dependency manifests.
+
+Update path
+===========
+
+Updating regular resources (such as SUIT manifests and images) alongside those responsible for device recovery in a single update increases the risk of placing the device in an unrecoverable state.
+To minimize this risk, recovery-dedicated SUIT manifests cannot be updated together with other manifests.
+
+Update path for elements not dedicated to SUIT recovery:
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_update_topology_for_non_recovery_elements.png
+
+   SUIT topology for the nRF54H20 SoC update path in the most complex predicted case for elements not dedicated to SUIT recovery.
+
+Update path for elements dedicated to SUIT recovery:
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_update_topology_for_recovery_elements.png
+
+   SUIT topology for the nRF54H20 SoC update path for elements dedicated to SUIT recovery.
+
+Recovery path
+=============
+
+Verification failures (for the SUIT manifest signature, SUIT manifest digest, and images) in the invocation path force the device to reboot into the recovery path.
+The recovery path is essentially an invocation path with a specific purpose: to download missing elements (such as images and manifests) and provide them to the SDFW as update candidates.
+
+.. warning::
+   To enable support for recovery, you must activate support for the application recovery manifest, see `Manifest provisioning information data structure`_.
+   If the device is in a state requiring recovery and the application recovery manifest is not activated or is damaged, the SDFW will boot from the root manifest.
```

Comment:
I must've been not precise enough, but we are getting there :slightly_smiling_face: 
```
If the device is in a state requiring recovery and the application recovery manifest is not activated the SDFW will boot from the root manifest.
If the application recovery manifest is activated, but the application recovery manifest is damaged, the SDFW will NOT boot any local images.
```